src/hearings/resolvers/judicial-user-search-response-resolver.resolve.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('Ref Data Resolver', () => {
     spyOn(service, 'getUsersData$').and.callThrough();
     service.resolve().subscribe((refData: JudicialUserModel[]) => {
       expect(service.getUsersByPanelRequirements$).toHaveBeenCalled();
-      expect(service.getUsersData$).toHaveBeenCalled();
+      expect(service.getUsersData$).not.toHaveBeenCalled();
```

Comment:
Isn't this cheating? Is that really what we expect? Or would it be better to to remove the line